Dockerfile.ci

```diff
@@ -26,7 +26,7 @@ ARG PYTHON_BASE_IMAGE
 ARG AIRFLOW_IMAGE_REPOSITORY="https://github.com/apache/airflow"
 
 # By increasing this number we can do force build of all dependencies
-ARG DEPENDENCIES_EPOCH_NUMBER="6"
+ARG DEPENDENCIES_EPOCH_NUMBER="7"
```

Comment:
I just realized this only needs to be bumped when _removing_ dependencies. Fix incoming.